docs/usage/index.md

```diff
@@ -0,0 +1,91 @@
+![Renovate banner](https://app.renovatebot.com/images/whitesource_renovate_660_220.jpg){ loading=lazy }
+
+# Renovate documentation
+
+Automated dependency updates.
+Multi-platform and multi-language.
+
+## Why use Renovate?
+
+<!-- markdownlint-disable list-marker-space -->
+<!-- prettier-ignore-start -->
+
+<div class="grid cards" markdown>
+
+-   :octicons-git-pull-request-24:{ .lg .middle } __Automatic updates__
+
+    ---
+
+    Get pull requests to update your dependencies and lock files.
+
+-   :octicons-calendar-24:{ .lg .middle } __On your schedule__
+
+    ---
+
+    Reduce noise by scheduling when Renovate creates PRs.
+
+-   :octicons-package-24:{ .lg .middle } __Works out of the box__
+
+    ---
+
+    Renovate finds relevant package files automatically, including in monorepos.
+
+-   :octicons-goal-24:{ .lg .middle } __How you like it__
+
+    ---
+
+    You can customize the bot's behavior with configuration files.
+
+-   :octicons-share-24:{ .lg .middle } __Share your configuration__
+
+    ---
+
+    Share your configuration with ESLint-like config presets.
+
+-   :octicons-sync-24:{ .lg .middle } __Out with the old, in with the new__
+
+    ---
+
+    Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions.
+
+-   :octicons-tools-24:{ .lg .middle } __Open source__
+
+    ---
+
+    Renovate is licensed under the [GNU Affero General Public License](https://github.com/renovatebot/renovate/blob/main/license).
+
+</div>
+
+<!-- prettier-ignore-end -->
+<!-- markdownlint-enable list-marker-space -->
+
+## Supported Platforms
+
+Renovate works on these platforms:
+
+- [GitHub (.com and Enterprise Server)](./modules/platform/github/readme.md)
```

Comment:
`build-docs` has a failing test in the "Test docs" step:

```
         # Subtest: index.md
        not ok 39 - index.md
          ---
          duration_ms: 9.508298
          location: 'file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:62:13'
          failureType: 'testCodeFailure'
          error: 'File not found: ./modules/platform/github/readme.md in index.md -> modules/platform/github/readme.md'
          code: 'ERR_ASSERTION'
          name: 'AssertionError'
          expected: true
          actual: false
          operator: '=='
          stack: |-
            checkNode (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:35:14)
            checkNode (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:47:7)
            checkNode (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:47:7)
            checkNode (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:47:7)
            checkNode (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:47:7)
            TestContext.<anonymous> (file:///home/runner/work/renovate/renovate/tools/docs/test/index.test.mjs:66:9)
            async Test.run (node:internal/test_runner/test:632:9)
            async Suite.processPendingSubtests (node:internal/test_runner/test:374:7)
          ...
```

Reply:
not `readme.md`, use `index.md`

Reply:
This is confusing! The file is called `readme.md` but the link should use `index.md`?

Here's the link to [`lib/modules/platform/github/readme.md`](https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/github/readme.md).

Reply:
yes, they got renamed when building docs. all modules readme files are required to be linked to index instead

Reply:
https://github.com/renovatebot/renovate/blob/2c66a362c182b10f0d4a57e7cd553e44359b5f33/tools/docs/manager.ts#L152

https://github.com/renovatebot/renovate/blob/2c66a362c182b10f0d4a57e7cd553e44359b5f33/tools/docs/manager.ts#L164